.github/kokoro/steps/git.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+echo
+echo "========================================"
+echo "Git log"
+echo "----------------------------------------"
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Git fetching tags"
+echo "----------------------------------------"
+git fetch --tags || true
```

Comment:
Are these `|| true` so that kokoro continues even if the commands fail? Maybe comment worthy that we want to continue even if the commands fail.